Test/MbrAnalysisTest.cs

```diff
@@ -294,8 +294,12 @@ public static void SpectralWriterTest()
 
             postSearchTask.Run();
 
-            var path = Path.Combine(TestContext.CurrentContext.TestDirectory, @"TestMbrAnalysisOutput\spectralLibrary.msp");
-            var testLibraryWithoutDecoy = new SpectralLibrary(new List<string> { path });
+            var path = Path.Combine(TestContext.CurrentContext.TestDirectory, @"TestMbrAnalysisOutput");
+            var list = Directory.GetFiles(path, "*.*", SearchOption.AllDirectories);
+            string matchingvalue = list.Where(p => p.Contains("spectralLibrary")).First().ToString();
+            var testLibraryWithoutDecoy = new SpectralLibrary(new List<string> { Path.Combine(path, matchingvalue) });
+            var libPath = Path.Combine(path, matchingvalue);
+
             var librarySpectra = testLibraryWithoutDecoy.GetAllLibrarySpectra().ToList();
```

Comment:
These changes look fine. How/where are spectral libraries saved now?

Reply:
It is saved in the same output folder but with a different name. The date will be appended to the name of spectral library, so you won't find it with "spectralLibrary.msp" only.